app/controllers/request-to-go-live/index/post.controller.js

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
 // Local dependencies
-const paths = require('../../../paths')
+const goLiveStageToNextPagePath = require('../go-live-stage-to-next-page-path')
 
 module.exports = (req, res) => {
-  // TODO: finish the UI implementation and logic
-  const externalServiceId = req.service.externalId
-  res.redirect(303, paths.requestToGoLive.index.replace(':externalServiceId', externalServiceId))
+  res.redirect(
+    303,
+    goLiveStageToNextPagePath[req.service.currentGoLiveStage].replace(':externalServiceId', req.service.externalId)
```

Comment:
This seems to follow the existing so this is merely a suggestions. To avoid the `replace` each time, `paths.requestToGoLive` could be changed to return a function which takes the `externalId` and returns the correct String using a template. eg.
```
requestToGoLive: {
    index: (externalId) => `/service/${externalId}/request-to-go-live`,
    organisationName: (externalId) => `/service/${externalId}/request-to-go-live/organisation-name`,
    chooseHowToProcessPayments: (externalId) => `/service/${externalId}/request-to-go-live/choose-how-to-process-payments`,
    agreement: (externalId) => `/service/${externalId}/request-to-go-live/agreement`
  }
```

and then in the POST controller...
```
goLiveStageToNextPagePath[req.service.currentGoLiveStage](req.service.externalId)
```

Reply:
Good idea! I started changing it and noticed that we use https://github.com/alphagov/pay-selfservice/blob/d221e6a8fe9b229c048849c0f6b37946ae753588/app/routes.js#L187 which relies on `String`s
As discussed one way to fix that will be to call all of the functions in `requestToGoLive` and have them to return the placeholder again, as discussed again with @danworth it is actually doing things more complicated because of `authenticatedPaths` so we are going to skip changing this